src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,80 @@
+package core.basesyntax;
+
+import core.basesyntax.enums.Figures;
+import core.basesyntax.models.Circle;
+import core.basesyntax.models.Figure;
+import core.basesyntax.models.IsoscelesTrapezoid;
+import core.basesyntax.models.Rectangle;
+import core.basesyntax.models.RightTriangle;
+import core.basesyntax.models.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static Random random = new Random();
+    private static ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int index = random.nextInt(Figures.values().length);
+        return Figures.values()[index].equals(Figures.CIRCLE) ? returnCircle() :
+                Figures.values()[index].equals(Figures.SQUARE) ? returnSquare() :
+                        Figures.values()[index].equals(Figures.RECTANGLE) ? returnRectangle() :
+                                Figures.values()[index].equals(Figures.ISOSCELES_TRAPEZOID)
+                                        ? returnIsoscelesTrapezoid() :
+                                        Figures.values()[index].equals(Figures.RIGHT_TRIANGLE)
+                                                ? returnRightTriangle()
+                                                : null;
+    }
+
+    public Figure getDefaultFigure() {
+        Circle circle = new Circle();
+        circle.setColor("WHITE");
+        circle.setRadius(10);
+        return circle;
+    }
+
+    public static Figure returnCircle() {

```

Comment:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/2d76b8ee-7c8e-403b-9d58-8187a29a91f2)

```suggestion
    public Figure getRandomCircle() {
```